README.md

```diff
@@ -144,30 +157,36 @@ You may provide basic auth credentials or access token as follows:
 - `JFROG_IDE_USERNAME` - JFrog Platform username
 - `JFROG_IDE_PASSWORD` - JFrog Platform password
 - `JFROG_IDE_ACCESS_TOKEN` - JFrog Platform access token
+
 </details>
 </details>
 
 **Notes:**
-* If your JFrog Platform instance uses a domain with a self-signed certificate, add the certificate to IDEA as described [here](https://www.jetbrains.com/help/idea/settings-tools-server-certificates.html).
-* From JFrog Xray version **1.9** to **2.x**, IntelliJ IDEA users connecting to Xray from IntelliJ are required to be granted the ‘View Components’ action in Xray.
-* From JFrog Xray version **3.x**, as part of the JFrog Platform, IntelliJ IDEA users connecting to Xray from IntelliJ require ‘Read’ permission. For more information, see [here](https://www.jfrog.com/confluence/display/JFROG/Permissions).
+
+- If your JFrog Platform instance uses a domain with a self-signed certificate, add the certificate to IDEA as described [here](https://www.jetbrains.com/help/idea/settings-tools-server-certificates.html).
+- From JFrog Xray version **1.9** to **2.x**, IntelliJ IDEA users connecting to Xray from IntelliJ are required to be granted the ‘View Components’ action in Xray.
+- From JFrog Xray version **3.x**, as part of the JFrog Platform, IntelliJ IDEA users connecting to Xray from IntelliJ require ‘Read’ permission. For more information, see [here](https://www.jfrog.com/confluence/display/JFROG/Permissions).
```

Comment:
The link still works, but it's a good opportunity to update it to the new one:

https://jfrog.com/help/r/jfrog-platform-administration-documentation/permissions

It's actually a good opportunity to update all of the old Confluence links 😬